src/core_modules/capture-core/components/Pages/New/RegistrationDataEntry/helpers/deriveAutoGenerateEvents.js

```diff
@@ -56,13 +53,13 @@ export const deriveAutoGenerateEvents = ({
                 const eventInfo = openAfterEnrollment
                     ? {
                         status: 'ACTIVE',
-                        occurredAt: convertFn(dateToUseInActiveStatus, dataElementTypes.DATE),
-                        scheduledAt: convertFn(dateToUseInActiveStatus, dataElementTypes.DATE),
+                        occurredAt: convertClientToServer(dateToUseInActiveStatus, dataElementTypes.DATE),
```

Comment:
Hey,
Do we need to do any conversion here at all? By looking at [useBuildEnrollmentPayload](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/DataEntries/EnrollmentRegistrationEntry/hooks/useBuildEnrollmentPayload.js#L94C43-L94C71), I can see that `enrolledAt` and `occurredAt` were already converted from [clientToServer](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/DataEntries/EnrollmentRegistrationEntry/hooks/useBuildEnrollmentPayload.js#L61) and passed to deriveAutoGenerateEvents. The `dateToUseInActiveStatus` value can either be enrolledAt or occurredAt and they are in the server format already. WDYT?
Thanks!

Reply:
Good catch @simonadomnisoru. Please remove the conversion here @superskip.

Reply:
Thanks, makes sense, will fix :+1: